chain-storage/src/lib.rs

```diff
@@ -79,8 +80,15 @@ pub struct Storage {
     db: Arc<dyn KeyValueDB>,
     /// tx to be committed
     current_tx: Option<DBTransaction>,
-    /// uncommitted values (for lookups during current block execution processing)
-    temp_tx_db: BTreeMap<(TxId, LookupItem), Vec<u8>>,
+}
+
+impl Get for Storage {
+    type Key = (u32, Vec<u8>);
+    type Value = Vec<u8>;
+    fn get(&self, key: &Self::Key) -> Option<Self::Value> {
+        let (col, key) = key;
+        self.db.get(*col, &key).unwrap()
```

Comment:
the unwrap is for IO errors? again, `expect` or a comment would be helpful

Reply:
Done